zio-kafka/src/main/scala/zio/kafka/consumer/internal/Runloop.scala

```diff
@@ -575,8 +536,9 @@ private[consumer] object Runloop {
     // Used for internal control of the runloop
     sealed trait Control extends Command
 
-    case object Poll        extends Control
-    case object StopRunloop extends Control
+    case object Poll         extends Control
+    case object StopRunloop  extends Control
+    case object StopFetching extends Control
```

Comment:
'Fetching' is not a word that we use yet. I suggest we stick to the terms that we have. E.g. something like:

```suggestion
    case object StopAllStreams extends Control
```